frontend/packages/ux-editor/src/containers/FormContext.tsx

```diff
@@ -88,9 +88,11 @@ export const FormContextProvider = ({ children }: FormContextProviderProps): JSX
   }, [handleComponentSave, handleContainerSave]);
 
   const handleEdit = useCallback((updatedForm: FormContainer | FormComponent): void => {
-    dispatch(setCurrentEditId(undefined));
-    setFormId(updatedForm?.id);
-    setForm(updatedForm);
+    if (updatedForm) {
```

Comment:
This is a bugfix and I suggest writing a unit-test for it. That would ensure that this bug does not get re-implemented later on.